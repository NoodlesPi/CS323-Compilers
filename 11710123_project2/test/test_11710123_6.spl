struct a{
    int b = 10000;
    int c = 0;

};
// type 15
struct a{
    int b = 10;
    int c = 1;

};
int main(){
    struct a app;
    app.b = 8000;
    app.c = app.b;
    // type 13
    app.d = 1000;
}