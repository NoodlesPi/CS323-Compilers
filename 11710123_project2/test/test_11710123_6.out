Error type 15 at Line 7: redefine the same structure type
Error type 14 at Line 17: accessing an undefined structure member
Error type 5 at Line 17: unmatching types on both sides of assignment operator (=)
